formula-android/src/main/java/com/instacart/formula/android/FragmentFlowStore.kt

```diff
@@ -122,14 +123,18 @@ class FragmentFlowStore @PublishedApi internal constructor(
                     val fragmentId = entry.key
                     val feature = (entry.value as? FeatureEvent.Init)?.feature
                     if (feature != null) {
-                        RxAction.fromObservable(feature) {
-                            feature.state.onErrorResumeNext {
-                                input.onScreenError(fragmentId.key, it)
-                                Observable.empty()
+                        val action = FeatureObservableAction(
```

Comment:
Created a custom action to move updates from the background to the main thread when needed.